packages/xo-server-sdn-controller/src/protocol/ovsdb-client.js

```diff
@@ -70,7 +70,7 @@ export class OvsdbClient {
   ) {
     if (
       this._adding.find(
-        elem => elem.id === network.uuid && elem.addr === remoteAddress
+        (elem) => elem.id === network.uuid && elem.addr === remoteAddress
```

Comment:
- refresh your branch on top of `master`
- make sure the dependencies are up-to-date (`yarn` at repo root)
- reformat files (`./node_modules/.bin/prettier --write '**/*.js'` at root)

Reply:
Done :)